harvard-lts/CURIOSity
# Fix unpublish

**Putting Edit Item form in new row.**
* * *

**Ticket**: [#294](https://github.com/harvard-lts/CURIOSity/issues/294)

# What does this Pull Request do?
When you edit an item on the detail page you cannot toggle the "Public" checkbox. This is because the item details are running over top of the edit form. See the screenshot below that uses the browser tools to show the item detail div overlapping with the edit form:

![ItemOverflow](https://user-images.githubusercontent.com/40801910/215538561-5cd8374b-a6ba-4876-8c4b-f73bba8e39ee.jpg)

This update puts the form in an additional div so you do not have the overflow problem and the "Public" checkbox can be toggled.

The new form looks like this:

![ItemNoOverflow](https://user-images.githubusercontent.com/40801910/215546697-87368517-cd12-453e-95de-c1c3abd636ff.jpg)

# How should this be tested?

A description of what steps someone could take to:
* Rebuild the container using the `fix-unpublish` branch
* Login as an administrator
* Edit any item in any exhibit and confirm you can toggle the "Public" checkbox.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@VanessaVenti @enriquediaz
